src/test/java/jenkins/plugins/logstash/LogstashIntegrationTest.java

```diff
@@ -231,4 +234,37 @@ public void enableGlobally() throws Exception
       assertThat(lastLine.getJSONArray("message").get(0).toString(),equalTo("Finished: SUCCESS"));
     }
 
+    @Test
```

Comment:
Wouldn't it be better to instead toggle the `milliSecondTimestamps` property instead of mocking `logstashConfiguration.getDateFormatter()`?

If not I think we can do with just a single test case since all it verifies is that `BuildData` is using the instance returned by `logstashConfiguration.getDateFormatter()` whatever the actual format

Reply:
Changed it so it calls the real methods